### PR TITLE
Adds CAN cooperation part of RC implementation 

### PR DESCRIPTION
**Pull request by dev-gh**
Adds CAN cooperation part of RC implementation  w/o RPC implemenation and w/o zone concept

### CODE REVIEW

**Comment by dev-gh** [comment]
@AKalinich-Luxoft @LuxoftAKutsan @LitvinenkoIra @AByzhynar Please review

File: src/components/can_cooperation/InteriorVehicleDataCapabilities.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "interiorVehicleDataCapabilities": [
+      {"moduleType": "CLIMATE"},
+      {"moduleType": "RADIO"},
+      {"moduleType": "CLIMATE"},
+      {"moduleType": "RADIO"},
+      {"moduleType": "CLIMATE"},
+      {"moduleType": "CLIMATE"}
```
Comment (AKalinich-Luxoft) [inline src/components/can_cooperation/InteriorVehicleDataCapabilities.json]: @dev-gh what a strange duplicates here?

File: src/components/can_cooperation/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,113 @@
+set(target "CANCooperation")
+set(install_destination ${CMAKE_BINARY_DIR}/bin/plugins)
+if (ENABLE_GCOV)
+  set(GCOV_FLAGS "-ftest-coverage -fprofile-arcs")
+else()
+  set(GCOV_FLAGS "")
+endif()
+
+set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++0x -Wall -Wno-deprecated-declarations -Werror ${GCOV_FLAGS}")
```
Comment (LuxoftAKutsan) [inline src/components/can_cooperation/CMakeLists.txt]: @dev-gh this cmake is not place for such logic ( it should be in main cmake)

File: src/components/can_cooperation/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  ${CMAKE_BINARY_DIR}/src/components/
+)
+
+set (CAN_SOURCE_DIR ${COMPONENTS_DIR}/can_cooperation/src)
+set (CAN_COMMANDS_DIR ${CAN_SOURCE_DIR}/commands)
+set (CAN_VALIDATORS_DIR ${CAN_SOURCE_DIR}/validators)
+set (CAN_STRUCT_VALIDATORS_DIR ${CAN_VALIDATORS_DIR}/struct_validators)
+
+set (SOURCES
+    ${CAN_SOURCE_DIR}/can_module.cc
```
Comment (LuxoftAKutsan) [inline src/components/can_cooperation/CMakeLists.txt]: @dev-gh lets try to use batch sources collecting

File: src/components/can_cooperation/InteriorVehicleDataCapabilities.json (hunk 1 above)
Comment (dev-gh) [reply src/components/can_cooperation/InteriorVehicleDataCapabilities.json]: @AKalinich-Luxoft 
Previously there were different zones. Fixed with ba3bb8f.

**Review by LuxoftAKutsan — CHANGES_REQUESTED** [review]

File: src/components/can_cooperation/CMakeLists.txt (hunk 2 above)
Comment (dev-gh) [reply src/components/can_cooperation/CMakeLists.txt]: @LuxoftAKutsan 
Done with 7ada254

File: src/components/can_cooperation/CMakeLists.txt (hunk 3 above)
Comment (dev-gh) [reply src/components/can_cooperation/CMakeLists.txt]: @LuxoftAKutsan 
Done with 7ada254

**Review by AKalinich-Luxoft — APPROVED** [review]

**Pull request merged**